jenkins/jobs/capm3-e2e-tests.pipeline

```diff
@@ -12,6 +12,9 @@ script {
 
     if ("${PROJECT_REPO_ORG}" == 'metal3-io' && "${PROJECT_REPO_NAME}" == 'project-infra') {
         ci_git_branch = (env.PULL_PULL_SHA) ?: (env.ghprbActualCommit) ?: "main"
+        ci_git_base = (env.PULL_BASE_REF) ?: "main"
+        // Fetch the base branch and the pullSha when running on project-infra PR
```

Comment:
you didnt use `pullSha`  here, but may be you should to make it consistent with other pipeline files ?

Reply:
thanks for noticing updated